### PR TITLE
Update jenkins.io ruleset

### PR DESCRIPTION
**Pull request by Jochen-A-Fuerbacher**

### CODE REVIEW

File: src/chrome/content/rules/jenkins.io.xml [hunk 1]
```diff
@@ -1,12 +1,13 @@
 <!--
 	Invalid certificate:
-	  nginx.azure.jenkins.io
-	  lettuce.jenkins.io
```
Comment (J0WI) [inline src/chrome/content/rules/jenkins.io.xml]: Why did you remove them?

Comment (Jochen-A-Fuerbacher) [reply src/chrome/content/rules/jenkins.io.xml]: Sublist3r didn't list them.

Comment (J0WI) [reply src/chrome/content/rules/jenkins.io.xml]: They're still up

**Comment by J0WI** [comment]
Thanks!
